scripts/chief_deploy.py

```diff
@@ -16,7 +16,9 @@
 
 # Setup venv path
 venv_bin_path = os.path.join(settings.VENV_DIR, 'bin')
-os.environ['PATH'] = venv_bin_path + os.pathsep + os.environ['PATH']
+os.environ['PATH'] = os.pathsep.join(
```

Comment:
This needs to be `os.environ['PATH'] = os.pathsep.join([venv_bin_path, '/usr/local/bin', os.environ['PATH']])`, note the square brackets